requirements/developer.txt

```diff
@@ -1,4 +1,3 @@
 black[jupyter]==22.3.0
-pre-commit>=2.12
```

Comment:
What happened to this?

Reply:
It's not used for anything currently, so I got rid of it.